### PR TITLE
modulesync 1.9.6; update all dependencies

### PR DESCRIPTION
**Pull request by bastelfreak**
modulesync 1.9.6

### CODE REVIEW

**Comment by bastelfreak** [comment]
Requires some releases...
* puppetlabs/java_ks https://tickets.puppetlabs.com/browse/MODULES-7745
* puppetlabs/mysql https://tickets.puppetlabs.com/browse/MODULES-7747
* puppetlabs/postgresql https://tickets.puppetlabs.com/browse/MODULES-7744

**Comment by bastelfreak** [comment]
currently waiting for the mysql release

File: .sync.yml [hunk 1]
```diff
@@ -3,5 +3,4 @@
   secure: "fH1bR1eBgA9tztMHtgdBSEOhNChC2YyoAczMzjUWgfDjinU1WOUPEprfV9upf6ZeQSTxAES36BBOrdxB6BVgdXlvLlCID+Lge1fTfi75ZX34q+eTadig/iZp3M0OveL2b5OmDv0cGbmMq8ywhgG+GPFuwPy7HF1Qjgul8UGNFOs="
   docker_sets:
     - set: centos7-64
-    - set: debian8-64
```
Comment (ekohl) [inline .sync.yml]: Could we update this to Debian 9 or is that a much bigger change?

Comment (bastelfreak) [reply .sync.yml]: sadly debian isn't supported at all. But I will have a look after this is merged.

**Pull request merged**